ui/admin/app/styles/app.scss

```diff
@@ -953,9 +952,29 @@
 
 // workers
 .workers {
-  .rose-dropdown-content {
-    max-height: 300px;
-    overflow: scroll;
+  .tags-list {
+    display: flex;
+    margin-bottom: 1rem;
+
+    .toolbar-refresher {
+      margin-left: auto;
+    }
+
+    .grouping {
+	      display: flex;
+	      align-items: center;
+	      gap: 0.2rem;
+	      padding: 0.5rem 1rem;
+	      text-transform: uppercase;
+	    }
+	
+	    li:has(.indentation) {
+	      padding-left: 2.25rem;
+	    }
+  }
+
+  .filters-applied {
+    margin-bottom: 1rem
```

Comment:
Alternatively, you could have just had 
```
  > div {
    margin-bottom: 1rem;
  }
```

Under `.workers` rather than two separate margin bottoms